lib/management.js

```diff
@@ -148,59 +183,65 @@ function applyReplicationState(conf, currentStateSerialized, db, cb) {
     }
 
     function deleteStream(streamId, cb) {
-        console.log('deleting obsolete stream', streamId);
+        logger.debug(`deleting obsolete stream ${streamId}`);
         const stream = currentState.streams[streamId];
         stream.enabled = false;
         putReplication(stream, err => {
             if (err) {
                 return cb(err);
             }
             delete currentState.streams[streamId];
-            saveCurrentReplicationState(db, currentState, cb);
+            return saveCurrentReplicationState(db, currentState, cb);
         });
     }
 
     function configureStream(stream, cb) {
         if (currentState.streams[stream.streamId]) {
-            if (!currentState.streams[stream.streamId].enabled && stream.enabled) {
-                console.log('enabling existing stream', stream.streamId);
+            if (!currentState.streams[stream.streamId].enabled &&
+            stream.enabled) {
+                logger.debug(`enabling existing stream ${stream.streamId}`);
                 putReplication(stream, err => {
                     if (err) {
                         return cb(err);
                     }
                     return streamConfigured(stream, cb);
                 });
-            } else if (currentState.streams[stream.streamId].enabled && !stream.enabled) {
-                console.log('disabling existing stream', stream.streamId);
+            } else if (currentState.streams[stream.streamId].enabled &&
+            !stream.enabled) {
+                logger.debug(`disabling existing stream ${stream.streamId}`);
                 putReplication(stream, err => {
                     if (err) {
                         return cb(err);
                     }
                     return streamConfigured(stream, cb);
                 });
             } else {
-                console.log('stream has expected state, doing nothing', stream.streamId, stream.enabled);
+                logger.debug('stream has expected state, doing nothing' +
+                    `${stream.streamId}, ${stream.enabled}`);
```

Comment:
I asked around and I think people told me that it's ok to not include an object/second arg for the logger - I'm also just converting the previous console.logs, so I don't want to add too much unnecessary info

Reply:
Oh yes, this looks fine. I had seen the comma without realizing it was part of the template string.

Reply:
We wrote this down in logging guidelines so that everyone writes logs the same way https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md#examples

Reply:
Agreed, since you are just converting it's fine, but for werelogs logs the message should not have "content", content should go in an attached object to be searchable.